Mw/widgets accessibility

michaelwood opened the pull request:
https://github.com/ThreeSixtyGiving/grantnav/issues/1056

Commit:
widgets: tabular_grants: Add funder into title tag if funder selected

Related: https://github.com/ThreeSixtyGiving/grantnav/issues/1056

Commit:
widgets: header: Make grantnav link contextual

Commit:
templates: base: Set the lang attribute to "en"

Related: https://github.com/ThreeSixtyGiving/grantnav/issues/1056

Commit:
widgets: header: Add title to logo link

Related: https://github.com/ThreeSixtyGiving/grantnav/issues/1056

codemacabre reviewed (dismissed):
Not part of your changes, but the `href` to `about:blank` seems like it would cause problems. Presumably you're handling this and rewriting the link elsewhere? I'm not sure what a screen reader would make of this if it encountered a link to a blank page.

michaelwood commented:
> Not part of your changes, but the `href` to `about:blank` seems like it would cause problems. Presumably you're handling this and rewriting the link elsewhere? I'm not sure what a screen reader would make of this if it encountered a link to a blank page.

Yeah that's a bit of an oddity apparently it gets set in JS https://github.com/ThreeSixtyGiving/grantnav/blob/live/grantnav/frontend/templates/widgets/grantnav_link_script.html

Commit:
frontend: templates: Remove or label when new tab is opened

To improve accessibility either remove or add text when a link is going
to open a new window.

codemacabre reviewed (approved):
lgtm!
